src/config.ts

```diff
@@ -48,9 +49,27 @@ export const config = reactive({
   customCollectionJsonPaths: createConfigRef(`${EXT_NAMESPACE}.customCollectionJsonPaths`, []),
 })
 
-export const customCollections = computed<readonly IconifyJSON[]>(() => {
-  return config.customCollectionJsonPaths.map(path => fs.readJSONSync(path))
```

Comment:
I would suggest resolving the path with `workspaceRoot`. So users could provide relative path for project custom config

```ts
  "iconify.customCollectionJsonPaths": [
    "./custom-collection.json", // project base
    "/absolute/path.json", // absolute
  ],
```

Also we could check the file existence to print log in the output channel instead of throwing hard error

Reply:
Will do; question - which workspace root?  Just the first one or check them in sequence?

I'm thinking of a setting set in the workspace file itself which is why I went this route.

Reply:
We could check for all workspace roots and only load existing ones.

Reply:
OK, made these changes!